love/src/redux/selectors/selectors.js

```diff
@@ -525,6 +525,15 @@ export const getM2ActuatorTable = (state) => {
   };
 };
 
+export const getM2PowerState = (state) => {
+  const subscriptions = ['event-MTM2-0-powerSystemState'];
+  const data = getStreamsData(state, subscriptions);
+  return {
+    powerType: data['event-MTM2-0-powerSystemState']?.[0].powerType?.value ?? 0,
+    powerState: data['event-MTM2-0-powerSystemState']?.[0].state?.value ?? 0,
+  };
+};
+
```

Comment:
If someone changes to use `event-MTM2-0-powerSystemState-update`, you would need to modify three places here.

Reply:
yep I agree, altohugh this is a design decision that comes from long time. Changing it would mean a full refactor of the `selectors.js` and probably other files :cry:.